datadog/resource_datadog_downtime.go

```diff
@@ -272,9 +276,10 @@ func buildDowntimeStruct(authV1 context.Context, d *schema.ResourceData, client
 	}
 	dt.SetScope(scope)
 	var tags []string
-	for _, mt := range d.Get("monitor_tags").([]interface{}) {
+	for _, mt := range d.Get("monitor_tags").(*schema.Set).List() {
 		tags = append(tags, mt.(string))
 	}
+	sort.Strings(tags)
```

Comment:
do we still need to sort them if we are using `schema.TypeSet` [1] ?

--
[1] https://www.terraform.io/docs/extend/schemas/schema-types.html#typeset